set.mm

```diff
@@ -30597,8 +30575,17 @@ generally appear in a single form (either definitional, but more often
     $( $j usage 'raleqbi1dv' avoids 'ax-8' 'ax-12'; $)
 
     $( Equality deduction for restricted existential quantifier.  (Contributed
-       by NM, 18-Mar-1997.)  (Proof shortened by Steven Nguyen, 5-May-2023.) $)
+       by NM, 18-Mar-1997.)  (Proof shortened by Steven Nguyen, 5-May-2023.)
+       Avoid ~ ax-8 .  (Revised by Wolf Lammen, 25-Sep-2024.) $)
     rexeqbi1dv $p |- ( A = B -> ( E. x e. A ph <-> E. x e. B ps ) ) $=
+      ( wceq wrex wn wral notbid raleqbi1dv ralnex 3bitr3g con4bid ) DEGZACDHZB
+      CEHZPAIZCDJBIZCEJQIRISTCDEPABFKLACDMBCEMNO $.
```

Comment:
Or prove a new `rexeqbidvv` from `raleqbidvv` and prove `rexeqbi1dv` from it.

Reply:
done